cmd/cca/verify_as_relyingparty.go

```diff
@@ -91,6 +95,15 @@ previous invocation to "evcli cca create" command.
 		"api-server", "s", "", "URL of the Veraison verification API",
 	)
 
+	relyingPartyIsInsecure = cmd.Flags().BoolP(
+		"insecure", "i", false, "Allow insecure connections (e.g. do not verify TLS certs)",
+	)
+
+	relyingPartyCerts = cmd.Flags().StringArrayP(
+		"ca-cert", "C", nil, "path to a CA cert that will be used in addition to system certs; may be specified multiple times",
```

Comment:
`pocli` has `E` as shorthand.  Would it be possible to align?  (Better UX, etc.)

Reply:
thanks!